src/components/Footer/Footer.js

```diff
@@ -2,20 +2,18 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 
 const Footer = () => (
-  <footer>
-    <div>
-      <p>
-        <span>
-          <FormattedMessage id="footer.iso" />
-        </span>
-        <span>
-          &copy; {new Date().getFullYear()} Doppler LLC.{' '}
-          <FormattedMessage id="footer.allRightReserved" />.{' '}
-          <a href="https://fromdoppler.com/privacidad">
-            <FormattedMessage id="footer.privacy" />
-          </a>
-        </span>
-      </p>
+  <footer className="footer-main">
+    <div className="footer-wrapper">
+      <span>
+        <FormattedMessage id="footer.iso" />
+      </span>
+      <span>
+        &copy; {new Date().getFullYear()} Doppler LLC.{' '}
+        <FormattedMessage id="footer.allRightReserved" />.{' '}
+        <a className="link--default" href="https://fromdoppler.com/privacidad">
```

Comment:
Privacy link has english counterpart `https://www.fromdoppler.com/en/privacy-policy` you should consider adding this to language file.

Reply:
@fcoronelmakingsense do not forget this.